invenio_admin/config.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Admin config."""
+
+from __future__ import unicode_literals
```

Comment:
Perhaps the ``u`` was unnecessary two lines below: https://github.com/inveniosoftware/invenio-admin/pull/2/files#diff-3fca13c63e83fb9792613cf0ea118207R24

As for ``unicode_literals`` I guess it is, given that we still support Python2 ?